tests/console/yast2_apparmor.pm

```diff
@@ -104,11 +104,13 @@ sub run {
         wait_screen_change { send_key 'down' };
         save_screenshot;
         send_key(is_pre_15() ? 'alt-t' : 'alt-c');
+        wait_still_screen(stilltime => 5);
         if (is_pre_15()) {
-            wait_screen_change { send_key 'tab' };
-            wait_screen_change { send_key 'end' };    # we need to search for recent toggled element at the of the list
+            send_key 'tab';
+            wait_still_screen(stilltime => 5);
```

Comment:
An explicit `assert_screen` is always wasting less time than a `wait_still_screen`. This call takes *at least* 5 seconds now even when not necessary. This test inefficiency accumulates over many calls and also many modules.

Reply:
Thanks for the tips, let me improve that!

Reply:
Created a new PR. Thanks!